test/svelte-scoped.test.ts

```diff
@@ -23,19 +25,40 @@ describe('svelte-scoped', () => {
     ],
   })
 
-  async function transform(code: string, { combine } = { combine: true }) {
-    return (await transformSvelteSFC(code, 'Foo.svelte', uno, { combine }))?.code
+  async function transform(code: string, { combine = true, format = true } = {}) {
+    const transformed = (await transformSvelteSFC(code, 'Foo.svelte', uno, { combine }))?.code
+    if (transformed && format) {
+      return prettier(transformed, {
+        parser: 'svelte',
+        plugins: [prettierSvelte],
+      })
```

Comment:
@antfu, wow, thank you for this! So much easier to read the test results. Initially I tried `prettierCSS` as seen elsewhere but obviously that didn't work since there's more than CSS in the test results.